content/vsphere-cpi.md

```diff
@@ -32,6 +32,19 @@ Schema for `cloud_properties` section used by manual network subnet:
 
 * **name** [String, required]: Name of the vSphere network. Example: `VM Network`.
 
+### Managed Networks ###
+Create T1 router and attach it to T0 router. It also creates virtual switch and attaches it to T1 router.
+So every subnet in network will have its own switch. Route advertisement is enabled on T1 router.
+
+Schema for `cloud_properties` section used by managed network subnet:
+
+* **name** [String, required]: Name of the vSphere network. Example: `Subnet 1`.
```

Comment:
Is this a logical name used inside of a deployment manifest; if so should it be something more akin to `subnet_1` or something without spaces/odd characters.

Reply:
Will update it